src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Functions/Processor.cs

```diff
@@ -75,7 +75,7 @@ IValidatorService validatorService
                     );
                     
                     logger.LogError($"{GetType().Name} function FAILED for : Datafile: " +
-                                    $"{message.DataFileName} : {e.Message}");
```

Comment:
I think if you `logger.LogError(e, message)` then we will get the stacktrace in application insights. Error is the first parameter

Reply:
Yep done - thanks